### DISCUSSION TITLE
HW 4 part 1

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#172] [Q&A]
Hello, 

How come the rest of my data isn't outputting?

![image](https://user-images.githubusercontent.com/116235922/211948365-3fdab676-c06e-4532-a2e3-a616c2f5ad68.png)

**Comment by Awasthi-Adi** [comment]
That code is outputting all of the data that it should.  If you are asking about HW4 Part 1; in case 1 with the spring mass system; I believe the output should be a singular matrix error; because there exists no unique solution.
